src/main/java/nextstep/optional/User.java

```diff
@@ -25,15 +29,18 @@ public static boolean ageIsInRange1(User user) {
         boolean isInRange = false;
 
         if (user != null && user.getAge() != null
-                && (user.getAge() >= 30
-                && user.getAge() <= 45)) {
+                && (user.getAge() >= MIN_FIND_AGE
+                && user.getAge() <= MAX_FIND_AGE)) {
             isInRange = true;
         }
         return isInRange;
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(User::getAge)
+                .filter(userAge-> userAge>=MIN_FIND_AGE)
+                .filter(userAge-> userAge <=MAX_FIND_AGE).isPresent();
```

Comment:
잘 작성해주셨네용! 

한가지 의견을 드리자면, 아래처럼 표현이 가능하네용 :) 
```suggestion
        return Optional.ofNullable(user)
                .map(User::getAge)
                .filter(age -> age >= MIN_FIND_AGE && age <= MAX_FIND_AGE)
                .isPresent();
```

Reply:
이런 바업ㅂ이 있는 줄 몰랐네요 ㅋㅋㅋㅋ 훨씬 깔끔해졌네요! 감사합니다! 이제 사다리 열심히 해볼게요! ㅋㅋㅋㅋ